docker/docker-compose.yaml

```diff
@@ -1,7 +1,7 @@
-version: "3"
+---
 services:
   adguardhome:
-    image: adguard/adguardhome:v0.107.48
+    image: adguard/adguardhome:v0.107.50
```

Comment:
I normally only update the version when I also update the underlying [adguard-client-go](https://github.com/gmichels/adguard-client-go) SDK, to ensure I catch any API changes appropriately.

Please revert to the current version.